modules/pytket-qiskit/pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -64,3 +83,71 @@ def required_predicates(self) -> List["Predicate"]:
 
     def default_compilation_pass(self, optimisation_level: int = 1) -> "BasePass":
         return self._ibmq.default_compilation_pass(optimisation_level)
+
+    @property
+    def _result_id_type(self) -> _ResultIdTuple:
+        return (str, int, str)
+
+    def process_circuits(
```

Comment:
why not just support postprocessing in AerBackend? Is the main reason inefficiency?

Reply:
Yes, mainly efficiency. The only advantage would be for AerBackends with a custom noise model. Perhaps we could consider this in the future but I think it would only really be useful if the gateset were customizable too.

Reply:
Not sure what you mean about customisable gateset?

Reply:
Well, the use case I see for an AerBackend with noise model is to simulate some actual device which will have a more limited gate set than the AerBackend. For IBM devices we have the IBMQEmulatorBackend; for others we'd need to do a custom rebase pass before running on the AerBackend, so the flow is less automatic anyway.